.clang-format

```diff
@@ -3,7 +3,7 @@ BasedOnStyle: Google
 ---
 Language:               Cpp
 Cpp11BracedListStyle:   true
-Standard:               c++11
+Standard:               Cpp11
```

Comment:
Reverted it to `Cpp11` as @songmeixu said he could install `clang-format 9.0.0` (the latest version we can install with `pip`).